app/src/main/res/layout-land/topic_practice_header_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.practice.practiceitemviewmodel.TopicPracticeHeaderViewModel" />
+  </data>
```

Comment:
`Ctrl + Alt + L` missing in this file too.

Reply:
Done